docs/client_retries.md

```diff
@@ -0,0 +1,320 @@
+# Client Side Retries
+Client Libraries use retries to handle unexpected, transient failures (i.e. server is temporarily unavailable).
+Multiple attempts, hopefully, will result in a successful response from the server.
+
+Default retry values are selected by the team operating the cloud service. These retry values are configured
+per RPC. A service *may* choose to only enable retries for a subset of RPCs. It is possible that each RPC for
+a service is configured differently.
+
+## Retry Parameters
+Client libraries have two types of retry parameters to configure:
+1. Retry Status Code: Set of failure status codes to retry on
+2. Retry Time/ Attempt Bounds: Configurable [RetrySettings](https://cloud.google.com/java/docs/reference/gax/latest/com.google.api.gax.retrying.RetrySettings) to define the bounds
+
+### RPC Retry Configurations
+Using Java-Asset v3.41.0 as an example, the default retry configurations are defined in the following places:
```

Comment:
The formatting here is a bit confusing as it is meant to be read as a list, but isn't formatted as one.

Consider adding bullets, and indenting each example with its bullet.

## For example...

Using Java-Asset v3.41.0 as an example, the default retry configurations are defined in the following places:
* Retry status codes...

    Example: 
    ```java
    java.logic;
    ```

* Retry parameters are...

    Example:
    ```java
    java.logic;
    ````

Reply:
Updating to be be indented in a list.